packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -979,6 +980,10 @@ export default {
                   });
                 }
               }
+            } else if (type === 'analysis-finding') {
+              const hash_v2 = object;
+              const finding = this.findings.filter(({ finding }) => finding.hash_v2 === hash_v2)[0];
```

Comment:
Use `this.findings.find(...)` instead. It'll only look for the first match instead of iterating the entire list. Also, `toListItem` looks to expect that it's input is never undefined, so it'd be best to handle that gracefully as well.

Reply:
Yup, good points, I made these changes :+1: